ci-operator/step-registry/baremetal/lab/post/add-nodes/baremetal-lab-post-add-nodes-commands.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+CLUSTER_NAME="${NAMESPACE}"
+DIR=/tmp/bmo
+mkdir -p ${DIR}
+
+master00_addr=$(yq e '.[] | select(.name == "master-00") | .bmc_address' ${SHARED_DIR}/hosts.yaml)
+master00_bootMacAddr=$(yq e '.[] | select(.name == "master-00") | .provisioning_mac' ${SHARED_DIR}/hosts.yaml)
+master00_username=$(yq e '.[] | select(.name == "master-00") | .bmc_user' ${SHARED_DIR}/hosts.yaml)
+master00_pass=$(yq e '.[] | select(.name == "master-00") | .bmc_pass' ${SHARED_DIR}/hosts.yaml)
+
+workera_addr=$(yq e '.[] | select(.name == "worker-a-00") | .bmc_address' ${SHARED_DIR}/hosts.yaml)
```

Comment:
we should do this in a loop so that we can address the case where more than 1 additional worker is requested.

Reply:
See https://github.com/openshift/release/blob/524564cec347aceaa462d41ac942b277077f9fd7/ci-operator/step-registry/upi/install/heterogeneous/upi-install-heterogeneous-commands.sh#L177-L186

Reply:
Done